2023/24xxx/CVE-2023-24010.json

```diff
@@ -1,18 +1,102 @@
 {
```

Comment:
CNA(s) of user (Alias) do not match owning CNA.

Reply:
Alias Robotics has robotics-related components within its scope and previously triaged and reported flaws concerning DDS technology and this vendor.